Sources/Supporting Files/tvOS/SessionTools_tvOS.h

```diff
@@ -0,0 +1,19 @@
+//
+//  SessionTools_tvOS.h
+//  SessionTools-tvOS
+//
+//  Created by Ryan Gant on 1/14/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
```

Comment:
Another copyright changed needed to use Bottle Rocket Studios

Reply:
Done.